tasks/release.js

```diff
@@ -51,6 +51,15 @@ module.exports = function (grunt) {
         platform.debName && platform.debName + '.sha1.txt'
       );
     }, [])
+    .reduce((files, file) => {
```

Comment:
I don't want two separate builds being created, though.  I want windows to be treated as a single build that simple gets renamed/zipped at the end, otherwise we'll need to smoketest/QA multiple windows builds.